File: ontrack-extension-git/src/main/java/net/nemerosa/ontrack/extension/git/model/GitChangeLogIssues.java
package net.nemerosa.ontrack.extension.git.model;

import lombok.Data;
import net.nemerosa.ontrack.extension.issues.model.IssueServiceConfigurationRepresentation;

import java.util.List;

@Data
public class GitChangeLogIssues {

    private final IssueServiceConfigurationRepresentation issueServiceConfiguration;
    private final List<GitChangeLogIssue> list;

}

File: ontrack-extension-git/src/main/java/net/nemerosa/ontrack/extension/git/model/GitChangeLogIssues.kt
package net.nemerosa.ontrack.extension.git.model

import net.nemerosa.ontrack.extension.issues.model.IssueServiceConfigurationRepresentation

class GitChangeLogIssues(
        val issueServiceConfiguration: IssueServiceConfigurationRepresentation,
        val list: List<GitChangeLogIssue>
)
